packaging/rhel/xrootd.spec.in

```diff
@@ -733,6 +733,10 @@ fi
 %{_libdir}/libXrdN2No2p-4.so
 %{_libdir}/libXrdOssSIgpfsT-4.so
 %{_libdir}/libXrdServer.so.*
+%{_libdir}/libXrdSsi-4.so.*
```

Comment:
Are we sure this should go to  server-libs, and not to a new package, .e.g. ssi-libs ?
Actually, since libXrdSsi is not versioned this fails as there are no libXrdSsi-4.si.*, there is only the libXrdSsi-4.so, which in turn is installed but not packaged.